polyhost/input/macos_helper.py

```
@@ -49,4 +50,20 @@ def set_language(self, lang, country):
             return False, msg
 
     def get_current_language(self):
-        return False, "Not Implemented" 
+        try:
+            result = subprocess.run(
+                ["defaults", "export", "com.apple.HIToolbox", "-"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)